AUTHORS

```diff
@@ -0,0 +1,10 @@
+# This is the official list of Forseti Security authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS files.
+# See the latter for an explanation.
+
+# Names should be added to this file as:
+# Name or Organization <email address>
+# The email address is not required for organizations.
+
+Google Inc.
+Spotify AB
```

Comment:
Do we add Denzel here as well?

Reply:
We can.  But according to the link below, it seems like he would have to initiate the request to be a copyright holder.
https://g3doc.corp.google.com/company/teams/opensource/releasing/authors.md?cl=head

Reply:
Makes sense. Removed the internal link, btw